examples/README.md

```diff
@@ -0,0 +1,5 @@
+-- Debug browser app
+
+1. `yarn --cwd ./packages/git watch` (to watch TS changes in src -> compiles to JS into the lib folder)
+2. `yarn --cwd ./examples/browser watch` (to watch for webpack. whenever the previous step modifies the packages in the lib folder, we regenerate the final bundle.js from the .js files for the app)
+3. `yarn --cwd ./examples/browser start` to start the app.
```

Comment:
@dhuebner I believe this file can be reverted?

Reply:
@vince-fugnitto
Sure. I just thought it might be useful for other contributors.

Reply:
@dhuebner thank you for the initiative, however, documentation regarding development is already covered in [developping.md](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md) which is also linked in the readme. If we are to improve the documentation regarding development, starting or debugging the examples it should be added to [developing.md](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md) as it would be easier to maintain and discover.